app/views/admin/organisations/_people_order.html.erb

```diff
@@ -1,10 +1,11 @@
 <%= render "govuk_publishing_components/components/summary_list", {
   title:,
-  edit: {
-    href: "#",
+  edit: { **(if @render_reorder
```

Comment:
I think we also need to add something about more than 1 role being present
```
if @render_reorder && organisation_roles.many?
```

<img width="549" alt="image" src="https://github.com/alphagov/whitehall/assets/42515961/f310a955-972c-4253-8009-2358d369f947">

Reply:
also this might be slightly easier to parse, totally optional as it's def working!

```
  edit: if @render_reorder && organisation_roles.many?
          {
            href: reorder_admin_organisation_people_path(@organisation, type:, title:),
            link_text: sanitize("Reorder <span class=\"govuk-visually-hidden\">#{title}</span>"),
            link_text_no_enhance: true
          }
        else
          {}
        end 
```

Reply:
Cool, thanks 👍